src/content/examples/hi/15_Math_And_Physics/03_Smoke_Particle_System/description.mdx

```diff
@@ -1,22 +1,11 @@
 ---
 featuredImage: "../../../images/featured/15_Math_And_Physics-03_Smoke_Particle_System-thumbnail.png"
-title: Smoke Particles
-one-line-description: Simulate smoke with a particle system.
+title: धुएँ के कण
+one-line-description: एक कण प्रणाली के साथ धुएं का अनुकरण करें।
 ---
 
 
-Smoke particle system demo, based on
-Dan Shiffman's original
-<a href="https://natureofcode.com/book/chapter-4-particle-systems/">example</a>
-for Processing.
-
-The code makes use of the
-<a href="https://p5js.org/reference/#/p5.Vector">p5.Vector</a>
-class, including the
-<a href="https://p5js.org/reference/#/p5/createVector">createVector()</a>
-function. The various calculations for updating particles'
-positions and velocities are performed with p5.Vector methods.
-
-The particle system is implemented as a
-<a href="https://p5js.org/reference/#/p5/class">class</a>, which contains an array of
-objects (of class Particle).
\ No newline at end of file
+स्मोक पार्टिकल सिस्टम डेमो, प्रसंस्करण के लिए डैन शिफमैन के मूल <a href="https://natureofcode.com/book/chapter-4-particle-systems/">उदाहरण</a> पर आधारित है। कोड का उपयोग करता है
```

Comment:
Some a tags have been translated and others haven't, do take a look across all files

Reply:
Yes, some of the tags have been translated as they were simple to understand and their meaning was the same in hindi. A few of the tags are not translated as the meaning was different in Hindi.